src/index.ts

```diff
@@ -0,0 +1,63 @@
+import { config } from './config/config';
+// need to be the first import and that the env variables are already loaded
+import * as apm from 'elastic-apm-node';
+apm.start({
+  serviceName: config.serviceName,
+  serverUrl: config.apm.host,
+  secretToken: config.apm.secretToken,
+  active: config.apm.active,
+});
+import promiseRetry = require('promise-retry');
+import server from './server';
+import * as mongoose from 'mongoose';
+import { log, LOG_LEVEL } from './helpers/logger';
+import { initIndex } from './search/elasticsearch';
+
```

Comment:
Connect to ES in this file:

1.  Create function to connect ES in ES.ts
2. Invoke this function here
1. 

1.